### DISCUSSION TITLE
Support for multiple libraries

### DISCUSSION THREAD

**Discussion by temeddix** [fzyzcjy/flutter_rust_bridge#943] [Ideas]
Hi. First of all, I would like to thank you for this amazing library. It's an amazing job.

I was wondering if this library could support generation of multiple libraries(.dll/.so) so that it can be properly used in a huge project. Right now, it seems possible to split API modules into `api_1.rs`, `api_2.rs`, but eventually it just gets compiled into a single library file. Wouldn't it be great to support multiple libraries, which is the same as multiple crates inside `/native`?

**Comment by fzyzcjy** [comment]
You are welcome! The suggestion looks quite reasonable, so feel free to PR :)

Also cc @dbsxdbsx who implemented the nice multi-file feature
